companies/w.md

```diff
@@ -16,7 +16,7 @@
   
 - Company: [Wasoko](https://wasoko.com)
 
-  - Founder: [@DanielYu]()
+  - Founders: [@DanielYu](https://twitter.com/danielyu) + [@JoshMRaine](https://twitter.com/joshmraine)
```

Comment:
Please include it in the respective js file

Reply:
@gray-adeyi  - Added the information inside src/techCompanies/orderByAlphabeth/w.js file.
Let me know if anything else is necessary.